src/queries/SUE/requestsWithServiceRequestId.tsx

```diff
@@ -1,12 +1,12 @@
 import _camelCase from 'lodash/camelCase';
 import _mapKeys from 'lodash/mapKeys';
 
-import { sueFetchObj, sueRequestsUrlWithServiceId } from '../../helpers';
+import { sueApi, sueRequestsUrlWithServiceId } from '../../helpers';
 
 export const requestsWithServiceRequestId = async (serviceRequestId: number) => {
-  const response = await (
-    await fetch(`${sueRequestsUrlWithServiceId(serviceRequestId)}`, sueFetchObj)
-  ).json();
+  const { sueFetchObj = {}, sueRequestsUrlWithServiceId = '' } = await sueApi(serviceRequestId);
```

Comment:
not `fetchSueEndpoints` here as well?

Reply:
Yeah, that's right, I missed that.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/1a6a8373aec308a3a1f1eeb83763b55eb027ee6f